test/units/testsuite-35.sh

```diff
@@ -360,6 +360,68 @@ EOF
     fi
 }
 
+setup_cron() {
+    # Setup test user and cron
+    useradd test || :
+    crond -s -n &
+    # Install crontab for the test user that runs sleep every minute. But let's sleep for
+    # 65 seconds to make sure there is overlap between two consecutive runs, i.e. we have
+    # always a cron session running.
+    crontab -u test - <<EOF
+RANDOM_DELAY=0
+* * * * * /bin/sleep 65
+EOF
+    # Let's wait (at most one interval plus 10s to accomodate for slow machines) for at least one session of test user
+    timeout 70s bash -c "while true; do loginctl --no-legend list-sessions | awk '{ print \$3 }' | grep -q test && break || sleep 1 ; done"
+}
+
+teardown_cron() {
+    set +e
```

Comment:
Please use subshell:
```
teardown_cron() {(
    set +ex

    ...
)}
```

Reply:
Actually, I think curly brackets are redundant if we have subshell, i.e. ().